test/cypress/components/filterable-list-control.js

```diff
@@ -0,0 +1,75 @@
+export class FilterableListControl {
+
+  filterableListControl() {
+    return cy.get( '.o-filterable-list-controls' );
+  }
+
+  filterableElement( selector ) {
+    return this.filterableListControl().get( selector );
+  }
+
+  filterableListElement( name ) {
+    return cy.get( `#o-filterable-list-controls_${ name }` );
+  }
+
+  openFilterableListControl() {
+    return cy.get( '#o-filterable-list-controls' )
+      .find( 'button' ).first().click( { force: true } );
+  }
+
+  filterItemName( name ) {
+    return this.filterableListElement( 'title' ).type( name );
+  }
+
+  filterFromDate( date ) {
+    return this.filterableListElement( 'from-date' ).type( date );
+  }
+
+  filterToDate( date ) {
+    return this.filterableListElement( 'to-date' ).type( date );
+  }
+
+  results() {
+    return cy.get( '.o-post-preview_content' );
+  }
+
+  firstResult() {
+    return this.results().first();
+  }
+
+  lastResult() {
+    return this.results().last();
+  }
+
+  openExpandable() {
+    return cy.get( '.o-filterable-list-controls .o-expandable' )
+      .click( { force: true } );
+  }
+
+  notification() {
+    return cy.get( '.o-filterable-list_notification' );
+  }
+
+  open() {
+    return this.openExpandable();
+  }
+
+  applyFilters() {
+    return cy.get( 'form[action="."]' ).submit();
+  }
+
+  clearFilters() {
+    cy.get( '.a-btn__warning' ).click( { force: true } );
+  }
+
+  showFilters() {
+    return cy.get( '[data-qa-hook="expandable"]' )
+      .find( 'button' ).first().click( { force: true } );
+  }
+
+  hideFilters() {
+    return cy.get( '[data-qa-hook="expandable"]' )
+      .find( 'button' ).last().click( { force: true } );
+  }
```

Comment:
The `qa-hook` data attributes are vestigial and should not be used. They could go away at any time. The buttons could be more directly accessed like so:

```suggestion
  showFilters() {
    return cy.get( 'o-expandable_target__collapsed' ).click();
  }

  hideFilters() {
    rreturn cy.get( 'o-expandable_target__expanded' ).click();
  }
```

I also think that if we `force` it, then it may not catch bugs that would cause the expandable to get in the wrong state at a given time.

Reply:
I do not see `o-expandable_target__collapsed` and `o-expandable_target__expanded` on `/about-us/blog/` page.
I am going to try to use `o-expandable_cue-open` and `o-expandable_cue-close` instead.

Reply:
I removed `data-qa-hook` attributes from `showFilters` and `hideFilters`